newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationPage.js

```diff
@@ -113,8 +143,20 @@ const PrivateAssetPackInformationPage = ({
 }: Props) => {
   const { id, name, sellerId } = privateAssetPackListingData;
   const { privateAssetPackListingDatas } = React.useContext(AssetStoreContext);
-  const { receivedAssetPacks, authenticated } = React.useContext(
-    AuthenticatedUserContext
+  const {
+    receivedAssetPacks,
+    authenticated,
+    profile,
+    limits,
+    assetPackPurchases,
+    getAuthorizationHeader,
+    onOpenLoginDialog,
+  } = React.useContext(AuthenticatedUserContext);
+  const { openCreditsPackageDialog, openCreditsUsageDialog } = React.useContext(
+    CreditsPackageStoreContext
+  );
+  const [selectedUsageType, setSelectedUsageType] = React.useState<string>(
+    privateAssetPackListingData.prices[0].usageType
```

Comment:
We always the first price sent by the server?

Reply:
Yes the server will return them by price increasing, so we pick the lowest (this allows changing it without risk in the future, as we keep picking the first one in the list)